components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.ts

```diff
@@ -1,5 +1,9 @@
 import {
-  Component, EventEmitter, Input, Output, OnChanges
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+  OnChanges
```

Comment:
While we probably do not have a company standard, I think most folks use this rule of thumb: if it fits on a line < 100 chars keep it on one line for succinctness.
(Lint is configured to complain above that length)